build-system/server.js

```diff
@@ -404,6 +404,18 @@ app.use('/examples/amp-fresh.amp.(min.|max.)?html', function(req, res, next) {
     next();
 });
 
+
+app.use('/impression-proxy/', function(req, res) {
+  assertCors(req, res, ['GET']);
```

Comment:
I thought about this.  I don't want to put a fake cookie here to get the ad server response. Is there a way to get the correct cookie here in proxy server ?(I don't think there should be due to security reason). If not I think I will stick with the fake ad response for now.